opentraveldata/optd_por_best_known_so_far.csv

```diff
@@ -12803,7 +12803,8 @@ SHA-C-1796236^SHA^31.22222^121.45806^SHA^
 SHB-A-6300326^SHB^43.5775^144.96^SHB^
 SHB-C-2129034^SHB^43.54849^144.97181^SHB^
 SHC-CA-7731491^SHC^14.08^38.28^SHC^
-SHD-CA-4785295^SHD^38.27^-78.9^SHD^
+SHD-A-4785295^SHD^38.26292^-78.90202^SHD^
+SHD-C-4785243^SHD^38.48512^-78.62502^SHD^
```

Comment:
Please be careful that the IATA code (SHD) should also be added on the corresponding Geonames POR.
I have done it for that one (http://www.geonames.org/4785243/shenandoah.html), but it is a general comment for future times.